src/PokeList.php

```diff
@@ -16,10 +16,17 @@ class PokeList {
      * Creates a new pokeList using a DBConnection
      * @param \PDO $connection The particular database object to use for retrieving the list of pokemon.
      */
-    public function __construct(\PDO $db)
+    public function __construct(\PDO $db, int $userId)
     {
-        $stmt = $db->prepare('SELECT `id`, `name`, `type_1`, `type_2` FROM `pokemon`');
-        $stmt->execute();
+        //$stmt = $db->prepare('SELECT `id`, `name`, `type_1`, `type_2` FROM `pokemon`');
+        $stmt = $db->prepare('
+          SELECT `pokemon`.`id`, `pokemon`.`name`, `pokemon`.`type_1`, `pokemon`.`type_2`, `new_status_table`.`seen_caught` 
+          FROM `pokemon` 
+          LEFT JOIN (SELECT `pokemon_id`, `seen_caught` 
+          FROM `status` 
+          WHERE `user_id` = ?) 
+          AS new_status_table ON `pokemon`.`id` = `new_status_table`.`pokemon_id`;');
```

Comment:
This makes your pokemon come out in the wrong order, as it will display pokemon whose status has been set over pokemon whose status has not.